### PR TITLE
refactor, chore(egen): updates dataset import process, adds gcfs library, other corrections from template

### PR DESCRIPTION
**Pull request by SumanthKasula99**
This PR consists of the following updates:

1. Adds code to copy the data into project's bucket, updates image paths in the DataFrame, and finally saves the modified DataFrame back to GCS as a CSV file.
2. Refactors the notebook according to the notebook template (removes boilerplate elements, renames REGION with LOCATION and re-structures the sections).
3. Adds link for Colab enterprise.
4. Removes the usage of future tense(will) according to the guidelines.
5. gcfs package is added at the installation step.

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/automl/automl_image_object_detection_online_prediction.ipynb [hunk 1]
```diff
@@ -86,12 +88,12 @@
         "\n",
         "The steps performed include:\n",
         "\n",
-        "- Create a Vertex `Dataset` resource.\n",
+        "- Create a Vertex Dataset resource.\n",
```
Comment (kittyabs) [inline notebooks/official/automl/automl_image_object_detection_online_prediction.ipynb:91]: - change: "Vertex Dataset"
- to: "Vertex AI dataset"

File: notebooks/official/automl/automl_image_object_detection_online_prediction.ipynb [hunk 2]
```diff
@@ -147,285 +156,318 @@
...
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "ad1138a125ea"
+        "id": "create_bucket"
       },
       "source": [
-        "**2. Local JupyterLab instance, uncomment and run:**"
+        "**Only if your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."
```
Comment (kittyabs) [inline notebooks/official/automl/automl_image_object_detection_online_prediction.ipynb:289]: Remove "Only"":
- "**If your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."

File: notebooks/official/automl/automl_image_object_detection_online_prediction.ipynb [hunk 3]
```diff
@@ -147,285 +156,318 @@
...
       "metadata": {
-        "id": "init_aip:mbsdk"
+        "id": "8c5a81fe0920"
       },
       "source": [
-        "## Initialize Vertex AI SDK for Python\n",
+        "#### Copying data between Google Cloud Storage Buckets\n",
         "\n",
-        "Initialize the Vertex AI SDK for Python for your project and corresponding bucket."
+        "In this step, you prevent access issues for the images in your original dataset. The code below extracts folder paths from image paths, constructs destination paths for Google Cloud Storage (GCS), copies images using gsutil commands, updates image paths in the DataFrame, and finally saves the modified DataFrame back to GCS as a CSV file."
```
Comment (kittyabs) [inline notebooks/official/automl/automl_image_object_detection_online_prediction.ipynb:389]: - change: "Google Cloud Storage (GCS)"
- to: "Cloud Storage".  (i.e., remove "Google" and don't use the abbreviation (GCS))

File: notebooks/official/automl/automl_image_object_detection_online_prediction.ipynb [hunk 4]
```diff
@@ -595,7 +637,7 @@
       "source": [
         "## Review model evaluation scores\n",
         "\n",
-        "After your model training has finished, you can review the evaluation scores for it using the `list_model_evaluations()` method. This method will return an iterator for each evaluation slice."
+        "After your model training has finished, you can review the evaluation scores for it using the `list_model_evaluations()` method. This method returns an iterator for each evaluation slice."
```
Comment (kittyabs) [inline notebooks/official/automl/automl_image_object_detection_online_prediction.ipynb:640]: - change: "it"
- to: "the model"

File: notebooks/official/automl/automl_image_object_detection_online_prediction.ipynb [hunk 5]
```diff
@@ -653,7 +695,7 @@
       "source": [
         "### Get test item\n",
         "\n",
-        "You will use an arbitrary example out of the dataset as a test item. Don't be concerned that the example was likely used in training the model -- we just want to demonstrate how to make a prediction."
+        "You use an arbitrary example out of the dataset as a test item. Don't be concerned that the examples were likely used in training the model -- the purpose here's to demonstrate how to make a prediction."
```
Comment (kittyabs) [inline notebooks/official/automl/automl_image_object_detection_online_prediction.ipynb:698]: Do not contract "here is".
- change: "the purpose here's to ..."
- to: "the purpose here is to ..."

**Review by kittyabs — COMMENTED** [review]
numerous edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**